crates/lang/codegen/src/generator/dispatch.rs

```diff
@@ -95,28 +95,24 @@ impl Dispatch<'_> {
         quote! {
             #[cfg(not(test))]
             #[no_mangle]
-            fn deploy() -> u32 {
-                ::ink_lang::DispatchRetCode::from(
-                    <#storage_ident as ::ink_lang::DispatchUsingMode>::dispatch_using_mode(
-                        ::ink_lang::DispatchMode::Instantiate,
-                    ),
+            fn deploy() {
+                <#storage_ident as ::ink_lang::DispatchUsingMode>::dispatch_using_mode(
+                    ::ink_lang::DispatchMode::Instantiate,
                 )
-                .to_u32()
+                .unwrap()
```

Comment:
Did you use an `unwrap` here intentionally over an `expect`?

Reply:
I felt that the error is sufficiently described the `Debug` implementation of `DispatchError`. I am open to write some message here if that is ink! convention, though.

Reply:
Would be great if you could replace the `unwrap`'s with `expects` then. We try to not use `unwrap`'s outside of tests.

Reply:
Done